Src/Support/Google.Apis.Auth.AspNetCore3.IntegrationTests/ClientInfo.cs

```diff
@@ -36,8 +36,16 @@ public static ClientInfo Load()
                 throw new InvalidOperationException($"Please set the {ClientSecretFilenameVariable} environment variable before running tests.");
             }
 
-            // This MUST be a "web" client ID, not an "installed app" client ID.
             var secrets = JObject.Parse(File.ReadAllText(clientSecretFilename))["web"];
+
+            // Check that this is a "web" client ID, not any other type of client ID like "installed app".
+            // The "web" element should have been present in the json so secrets value shouldn't be null.
+            if (secrets is null)
+            {
+                throw new ArgumentException(
```

Comment:
I think this should be InvalidOperationException - it's not an argument to this method. (So remove the second constructor argument too.)

Reply:
Yes, done.